src/screens/Settings/Account.jsx

```diff
@@ -128,6 +171,58 @@ export default function Account() {
         <GoogleSignin text="Connect to Google" />
       )}
 
+      <Title s2 className={styles.span2}>
+        Time Zone
+      </Title>
+      <Text className={styles.span2}>
```

Comment:
Suggestion for new text:
The video call times in your email notifications are set to *Europe/Berlin (CEST)*. You can change your timezone here.
All dates and times you see at Upframe are displayed in your current timezone by default.

Reply:
Can we change that to "on Upframe" or "on upframe.io"? I think "at Upframe" would be grammatically incorrect.
And I would also change it to "The video call times … are displayed in _Central European Summer Time_", since setting a time to a location doesn't really make sense.

Reply:
Let's go with on Upframe 👍 
Next time, when it comes to my grammar mistakes, just go ahead and merge it!